### DISCUSSION TITLE
Exporting CI/PI/VPC

### DISCUSSION THREAD

**Discussion by RobinM92** [Open-Systems-Pharmacology/Forum#1336] [Questions & Problems]
Dear community, 
As most people, I use post-processing software (R in this case) to convert PKSim or MoBi results into figures for e.g. a publication or a report. For single simulations, this is quite straightforward as these can be easily exported to csv and read into R again. However when I try to do the same for the results of a parameter identification step, e.g. to plot the CI or VPC, this can only be done to xlsx (in mobi) and the resulting file only shows the observed data and (in separate sheets) the median predictions. Am I missing a straightforward way to get e.g. the percentiles as well?

Thank you!
Kind regards,
Robin

**Comment by RobinM92** [comment]
Does anyone have an idea? I feel this should be simple but I am not finding how to do this.

**Reply by msevestre** [reply]
Can you explain what you are trying to export? Are you also suggesting that the export you found is only in MoBi?

**Reply by RobinM92** [reply]
Hi @msevestre ,

When exporting any simulation which is not just a deterministic profile to excel for post-processing in R, only a mean profile gets exported. E.g. when trying to export a VPC, a PI, a CI or a population simulation by right-clicking on the graph and selecting 'export-to-excel', a file is created which contains any simulated mean profile, but nothing else. Maybe I am doing it wrong but I would expect at least the simulated percentiles to be outputted as well?

**Reply by msevestre** [reply]
I think this was simply never requested. 
This is a default export inherent to each chart component that needs to be specialized for each use case. I guess, this was never done here. 
So at this stage, thee is probably not easy way to export what you need. 

Would you please be so kind and create a feature request in the PKSim repository explaining what you would expect to be exported? Ideally an example of what would the ideal export structure look like and a project so that we don't have to spent too much time creating a context for the implementation

**Comment by prvmalik** [comment]
I'm not sure I understand why such output would be needed. 

In PBPK models, all parameters should be uniquely and precisely identifiable - very little RSE. This should be shown through multiple optimizations from random start values ending at the exact solution. 

For population simulations, you do not need complex statistics or confidence intervals. Confidence intervals for the plasma concentrations? Just simulate with N>10000 to trend to the exact median, 5th and 95th of the simulation. Output the 5th and 95th when you edit the time profile and it comes out on excel.

**Reply by RobinM92** [reply]
Hi Paul,
I agree that CIs are typically redundant (should be very small), but it is still rather strange that something that is calculated within the software cannot be exported, in my opinion.
The PIs, which are to my understanding also based on the variability in the data could be useful in post-processing, and the combination of the two, VPC (which I feel is a bit of an unfortunate name given our field) should also be exportable.

I will double check whether I can export the percentiles of a population simulation following your description, that would already be very helpful! 

Thanks for the input!

**Reply by wilbertdew** [reply]
Hi Paul,

I think there are some exceptions where RSE is significant, e.g. for molecule properties or for some less impactful parameters like Lymph flow in the large molecule model. This has CV values in the different tissues up to 91% in the Niederalt publication and given the low impact I dont think this is necessarily a problem. What do you think?
Best
Wilbert
